fog/ledger/enclave/api/src/lib.rs

```diff
@@ -1,26 +1,27 @@
 // Copyright (c) 2018-2021 The MobileCoin Foundation
 
-//! APIs for MobileCoin Ledger Service Enclaves
+//! APIs for MobileCoin Ledger Service Enclave
 
 #![no_std]
 
```

Comment:
can we add `#![deny(missing_docs)]` and put doc strings on all of these types? it is important to document API types